### PR TITLE
Fix tests baremetal-cucushift-ipi

### PR DESCRIPTION
**Pull request by liangxia**
Scenario failures should not block fix in job level.

### CODE REVIEW

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.9.yaml [hunk 1]
```diff
@@ -29,6 +29,9 @@ tests:
   interval: 6h
   steps:
     cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG: |
```
Comment (JianLi-RH) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.9.yaml:33]: `|` or `|-`?

Comment (liangxia) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-4.9.yaml:33]: It's `|`,  not `|-`
```
[lxia@lxiabox release]$ grep -r 'DEVSCRIPTS_CONFIG.*|' | wc -l
560

[lxia@lxiabox release]$ grep -r 'DEVSCRIPTS_CONFIG.*|-' | wc -l
0
```

File: ci-operator/step-registry/cucushift/baremetal/ipi/cucushift-baremetal-ipi-commands.sh [hunk 2]
```diff
@@ -4,10 +4,16 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+if [ -f "${SHARED_DIR}/kubeconfig" ] ; then
+    export KUBECONFIG=${SHARED_DIR}/kubeconfig
```
Comment (jhou1) [inline ci-operator/step-registry/cucushift/baremetal/ipi/cucushift-baremetal-ipi-commands.sh:8]: Why do we have to export KUBECONFIG here?

Comment (liangxia) [reply ci-operator/step-registry/cucushift/baremetal/ipi/cucushift-baremetal-ipi-commands.sh:8]: @jhou1  That comes from an conversation with metal-ipi owners, https://coreos.slack.com/archives/CFP6ST0A3/p1634219671275200?thread_ts=1634216796.272000&cid=CFP6ST0A3

**Comment by jhou1** [comment]
I see tests have been finished, several scenarios failures for us to keep debugging. This won't block the fixes to the steps and job configuration.
/lgtm
/approve

**Pull request merged**